posthog/utils.py

```diff
@@ -182,7 +183,7 @@ def get_git_commit() -> Optional[str]:
     """
 
     try:
-        return subprocess.check_output(["git", "rev-parse", "--short", "HEAD"]).decode("utf-8").strip()
```

Comment:
Why add a new dependency (`gitpython`) instead of using the existing way? Both `subprocess` and `gitpython` require system `git`, so I'd prefer to avoid a whole new dependency for this single thing.